internal/generated/snippets/speech/apiv1p1beta1/AdaptationClient/CreateCustomClass2/main.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by cloud.google.com/go/internal/gapicgen/gensnippets. DO NOT EDIT.
+
+// [START speech_v1p1beta1_generated_Adaptation_CreateCustomClass_sync_2]
+
+package main
+
+import (
+	"context"
+	"log"
+
+	speech "cloud.google.com/go/speech/apiv1p1beta1"
+	"google.golang.org/api/option"
+	speechpb "google.golang.org/genproto/googleapis/cloud/speech/v1p1beta1"
```

Comment:
The API version specified on the config is v1, so this should be for v1, not for the beta.

Reply:
Fixed, thank you!